ppml/trusted-big-data-ml/python/docker-graphene/test-suites/kms-e2e-example-v2.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# At /ppml/trusted-big-data-ml
+
+#set -x
+# Modify Below Variables According To Your Environment
+KMS_SERVER_IP=192.168.0.112
+INPUT_DB_PATH=/ppml/trusted-big-data-ml/1m.db
+LOCAL_IP=192.168.0.112
+
+# Step 1. Generate Keys And Use Them to Encyypt Files Outside SGX With KMS
```

Comment:
step.1 generate primary key and data key 
step.2 encrypt input files  [in own trusted env]
step.3 submit spark job to decrypt inputs, do business logic, and encrypt to outputs [Driver in SGX, Executor in SGX]
step.4 decrypt output fields [in own trusted env]

separated scripts

Reply:
step.3 should run with spark local and SGX-K8S